constants/quality_controllers.go

```diff
@@ -0,0 +1,35 @@
+// constants holds all the constants shared between packages
+package constants
+
+const (
+	// Canu is the name of the program GenoMagic uses to perform error correction
+	Canu = "canu"
+
+	// Trimmomatic is the name of the program GenoMagic uses to perform adapter trimming
+	Trimmomatic = "trimmomatic"
```

Comment:
Let's change "Trimmomatic" to "Porechop", because we're no longer using Trimmomatic since Porechop is more sophisticated and applies to ONT. I am later going to ensure nanopore and illumina are trimmed with their respective software: (Issue #73)